www/app/blog/2025-05-09-uuidv7.mdx

```diff
@@ -0,0 +1,176 @@
+export const metadata = {
+  title: "UUIDv7 Comes to PostgreSQL 18",
+  authors: ["gwenshap"],
+  image: "2025-05-09-uuidv7/cover.png",
+  sizzle:
+    "PostgreSQL 18 adds native support for UUIDv7. A sortable UUID variant that plays nicely with indexes. Here's what that means for you.",
+  tags: ["postgres", "performance"],
+};
+
+PostgreSQL 18 is on the horizon, with beta testing now underway. Among the many improvements in this release is support 
+for UUIDv7. A UUID variant that is sortable and plays nicely with indexes. In this post, we'll discuss UUIDs in general, why UUIDv7 is so useful and how you'll want to use it in Postgres.
+
+## PostgreSQL 18
+
+[PostgreSQL 18 beta 1 was released](https://www.postgresql.org/about/news/postgresql-18-beta-1-released-3070/) few days ago. The release is packed with new features, improvements and bug fixes.
+As usual, the community is encouraged to try it out and report issues, with the goal of shipping a high quality release in September.
+
+The highlights of the release include:
+
+- Async I/O (with io_uring) — 2-3x speedups on seq scans, vacuums
+- Skip scan + smarter OR/IN optimizations
```

Comment:
nit: index skip scan I think ? and I think only for btree (I didn't look at that feature yet so I might be wrong)

Reply:
Thats right. The release notes say: "PostgreSQL 18 adds support for using "skip scan" lookups on multicolumn B-tree indexes, which can result in faster execution times for queries that omit a "=" condition on one or more prefix index columns"